docker-compose-ci.yml

```diff
@@ -5,6 +5,10 @@ services:
     image: postgres:9.5-alpine
     logging:
       driver: "none"
+    ports:
+      - "5432:5432"
+    environment:
+      - POSTGRES_HOST_AUTH_METHOD=trust
```

Comment:
I was running into: https://gitlab.com/gitlab-org/gitlab/-/issues/206944 which mentions that `POSTGRES_HOST_AUTH_METHOD=trust` is now required